concept/inquirer.go

```diff
@@ -58,23 +59,23 @@ func (n *NeoInquirer) Inquire(candidates []string, tid string) []*Worker {
 		workers = append(workers, worker)
 	}
 	go func() {
-		log.WithField("transaction_id", tid).Infof("Starting reading concepts from Neo: %v", candidates)
+		n.Log.WithField("transaction_id", tid).Infof("Starting reading concepts from Neo: %v", candidates)
```

Comment:
Instead of writing `WithField("transaction_id", tid)` everywhere we can create `logEntry` object that we can use to log with
```suggestion
		entry := n.Log.WithTransactionID(tid)
		entry.Infof("Starting reading concepts from Neo: %v", candidates)
```

Reply:
I would change on line 66
```
if err != nil {
  logEntry.WithError(err).Errorf("Error by reading %v concept type from Neo", worker.ConceptType)
  worker.Errch <- err
  continue
}
```

and on line 71
```
if !found {
  err = fmt.Errorf("Reading %v concept type from Neo returned empty result", worker.ConceptType)
  logEntry.Error(err)
  worker.Errch <- err
  continue
}
```